test/messages.jl

```diff
@@ -17,167 +15,147 @@ using HTTP.MessageRequest: unknown_length
 using JSON
 
 @testset "HTTP.Messages" begin
+    req = Request("GET", "/foo", ["Foo" => "Bar"])
+    res = Response(200, ["Content-Length" => "5"]; body="Hello", request=req)
 
-    @test bodylength(7) == unknown_length
-    @test bodylength(UInt8[1,2,3]) == 3
-    @test bodylength(view(UInt8[1,2,3], 1:2)) == 2
-    @test bodylength("Hello") == 5
-    @test bodylength(SubString("World!",1,5)) == 5
-    @test bodylength(["Hello", " ", "World!"]) == 12
-    @test bodylength(["Hello", " ", SubString("World!",1,5)]) == 11
-    @test bodylength([SubString("Hello", 1,5), " ", SubString("World!",1,5)]) == 11
-    @test bodylength([UInt8[1,2,3], UInt8[4,5,6]]) == 6
-    @test bodylength([UInt8[1,2,3], view(UInt8[4,5,6],1:2)]) == 5
-    @test bodylength([view(UInt8[1,2,3],1:2), view(UInt8[4,5,6],1:2)]) == 4
-    @test bodylength(IOBuffer("foo")) == 3
-    @test bodylength([IOBuffer("foo"), IOBuffer("bar")]) == 6
-
-    @test bodybytes(7) == UInt8[]
-    @test bodybytes(UInt8[1,2,3]) == UInt8[1,2,3]
-    @test bodybytes(view(UInt8[1,2,3], 1:2)) == UInt8[1,2]
-    @test bodybytes("Hello") == bytes("Hello")
-    @test bodybytes(SubString("World!",1,5)) == bytes("World")
-    @test bodybytes(["Hello", " ", "World!"]) == UInt8[]
-    @test bodybytes([UInt8[1,2,3], UInt8[4,5,6]]) == UInt8[]
+    protocols = ["http", "https"]
+    http_reads = ["GET", "HEAD", "OPTIONS"]
+    http_writes = ["POST", "PUT", "DELETE", "PATCH"]
+
+    @testset "Body Length" begin
+        @test bodylength(7) == unknown_length
+        @test bodylength(UInt8[1,2,3]) == 3
+        @test bodylength(view(UInt8[1,2,3], 1:2)) == 2
+        @test bodylength("Hello") == 5
+        @test bodylength(SubString("World!",1,5)) == 5
+        @test bodylength(["Hello", " ", "World!"]) == 12
+        @test bodylength(["Hello", " ", SubString("World!",1,5)]) == 11
+        @test bodylength([SubString("Hello", 1,5), " ", SubString("World!",1,5)]) == 11
+        @test bodylength([UInt8[1,2,3], UInt8[4,5,6]]) == 6
+        @test bodylength([UInt8[1,2,3], view(UInt8[4,5,6],1:2)]) == 5
+        @test bodylength([view(UInt8[1,2,3],1:2), view(UInt8[4,5,6],1:2)]) == 4
+        @test bodylength(IOBuffer("foo")) == 3
+        @test bodylength([IOBuffer("foo"), IOBuffer("bar")]) == 6
+    end
 
+    @testset "Body Bytes" begin
+        @test bodybytes(7) == UInt8[]
```

Comment:
Looks like we already have test for `bodybytes`, so I say we just remove the `Body.jl` file.